plane_spotter/twitter.py

```diff
@@ -83,6 +83,9 @@ def unusual_activity_challenge(self, log=logger):
         except selenium.common.exceptions.NoSuchElementException:
             log.info("unusual activity challenge not present")
             return
+        except selenium.common.exceptions.TimeoutException:
+            log.info("unusual activity challenge timed out")
+            return
```

Comment:
I think it will always return a TimeoutException if the field doesn't exist, so maybe we can remove the NoSuchElementException clause altogether.

Reply:
yep that's been my experience in testing, just left the old exceptions around just in case. will remove.